modules/backend/assets/js/october.tabformexpandcontrols.js

```diff
@@ -160,4 +160,209 @@
     }
 
     $.oc.tabFormExpandControls = TabFormExpandControls
-}(window.jQuery);
\ No newline at end of file
+
+}(window.jQuery);
+
+
+/*
+ * Auto update WAI-ARIA when a user updates the tabs
+ */
+ $(document).ready(function(){
+
+     /* Master Tabs */
+
+     // if no selected => select first
+     if ($('.master-tabs a[aria-selected]').length == 0) {
+         $('.master-tabs a:first').attr('aria-selected', 'true');
+         $('.master-tabs a:first').attr('tabindex', '0');
+     }
+
+     /* click on a link */
+     $('.master-tabs a').click(function() {
+         // remove aria selected on all links + remove focusable
+         $('.master-tabs a').attr('aria-selected', 'false');
+
+         // add aria selected on $this + focusable
+         $('.master-tabs a.focus-visible').attr('aria-selected', 'true');
+
+         // Important - Must be set to true for October to set class="active" in the <li>
+         return true;
+     });
+
+     $('body').on('keydown','.master-tabs', function(event) {
+         // strike up or left in the tab
+         if (event.keyCode == 37 || event.keyCode == 38) {
+             // find previous tab
+             // if we are on first => activate last
+             $activated = $('.master-tabs a[aria-selected="true"]').parent();
+             if ($activated.is('.master-tabs li:first-child')) {
+                 $('.master-tabs li:last-child a').click().focus();
+             } else {
+                 // else activate previous
+                 $activated.prev().children('.master-tabs a').click().focus();
+             }
+             event.preventDefault();
```

Comment:
@ayumihamsaki any reason why `preventdDefault` is at the bottom of the function? I believe you'll have an odd behavior in a few browsers, not based on the chrome.

Reply:
Fixed.